src/main/kotlin/fr/social/gouv/agora/usecase/reponseConsultation/InsertReponseConsultationUseCase.kt

```diff
@@ -13,20 +15,26 @@ import org.springframework.stereotype.Service
 
 @Service
 class InsertReponseConsultationUseCase(
+    private val contentSanitizer: ContentSanitizer,
     private val consultationPreviewAnsweredRepository: ConsultationPreviewAnsweredRepository,
     private val insertReponseConsultationRepository: InsertReponseConsultationRepository,
     private val consultationResponseRepository: GetConsultationResponseRepository,
     private val questionRepository: QuestionRepository,
     private val insertConsultationResponseParametersMapper: InsertConsultationResponseParametersMapper,
     private val consultationPreviewPageRepository: ConsultationPreviewPageRepository,
 ) {
+    companion object {
+        private const val AUTRE_QUESTION_MAX_LENGTH = 200
```

Comment:
In anglais por favor in order pour avoid el Franglais ?

Reply:
done